parsers/lib/quality.py

```diff
@@ -200,5 +200,14 @@ def validate_production(obj: Dict[str, Any], zone_key: ZoneKey) -> None:
                 "Couldn't find emission factor for '%s' in '%s'. Maybe you misspelled one of the production keys?"
                 % (key, zone_key)
             )
+    if "production" in obj.keys():
+        if all(
+            (val== 0)
+            or (val is None)
+            for val in obj.get("production",{}).values()
+        ):
+            raise ValidationError(
+                f"{zone_key}, {obj.get('datetime')}: unrealistic datapoint, all production values are 0.0 MW or null"
+            )
```

Comment:
I don't think we can do it like this, we have a few zones where it's expected to be all 0 at times when they only rely on imports, like Åland (AX).

Reply:
hmmmm annoying. it does make AX fail ...

Reply:
There should be a config in the backend config that has the list of allowed 0 production zones. Maybe we could just exclude those (and move the config here)?

Reply:
@VIKTORVAV99, do you have any idea of other zones that would also be like this?

Reply:
ok I decided to go about this the other way around and to add validation at the parser level

Reply:
> @VIKTORVAV99, do you have any idea of other zones that would also be like this?

There are a few other zones, mostly islands that have a interconnection. But I think the parser level is the way to go for this particular zone.